packages/adapter-kysely/src/index.ts

```diff
@@ -0,0 +1,435 @@
+ *     .execute()
+ *
+ *   await db.schema
+ *     .createTable("Account")
+ *     .addColumn("id", "uuid", (col) =>
+ *       col.primaryKey().defaultTo(sql`gen_random_uuid()`)
+ *     )
+ *     .addColumn("userId", "uuid", (col) =>
+ *       col.references("User.id").onDelete("cascade").notNull()
+ *     )
+ *     .addColumn("type", "text", (col) => col.notNull())
+ *     .addColumn("provider", "text", (col) => col.notNull())
+ *     .addColumn("providerAccountId", "text", (col) => col.notNull())
+ *     .addColumn("refresh_token", "text")
+ *     .addColumn("access_token", "text")
+ *     .addColumn("expires_at", "bigint")
+ *     .addColumn("token_type", "text")
+ *     .addColumn("scope", "text")
+ *     .addColumn("id_token", "text")
+ *     .addColumn("session_state", "text")
+ *     .addColumn("oauth_token_secret", "text")
+ *     .addColumn("oauth_token", "text")
+ *     .execute()
+ *
+ *   await db.schema
+ *     .createTable("Session")
+ *     .addColumn("id", "uuid", (col) =>
+ *       col.primaryKey().defaultTo(sql`gen_random_uuid()`)
+ *     )
+ *     .addColumn("userId", "uuid", (col) =>
+ *       col.references("User.id").onDelete("cascade").notNull()
+ *     )
+ *     .addColumn("sessionToken", "text", (col) => col.notNull().unique())
+ *     .addColumn("expires", "timestamptz", (col) => col.notNull())
+ *     .execute()
+ *
+ *   await db.schema
+ *     .createTable("VerificationToken")
+ *     .addColumn("identifier", "text", (col) => col.notNull())
+ *     .addColumn("token", "text", (col) => col.notNull().unique())
+ *     .addColumn("expires", "timestamptz", (col) => col.notNull())
+ *     .execute()
+ *
+ *   await db.schema
+ *     .createIndex("Account_userId_index")
+ *     .on("Account")
+ *     .column("userId")
+ *     .execute()
+ *
+ *   await db.schema
+ *     .createIndex("Session_userId_index")
+ *     .on("Session")
+ *     .column("userId")
+ *     .execute()
+ * }
+ *
+ * export async function down(db: Kysely<any>): Promise<void> {
+ *   await db.schema.dropTable("Account").ifExists().execute()
+ *   await db.schema.dropTable("Session").ifExists().execute()
+ *   await db.schema.dropTable("User").ifExists().execute()
+ *   await db.schema.dropTable("VerificationToken").ifExists().execute()
+ * }
+ * ```
+ * > This schema is adapted for use in Kysely and is based upon our main [schema](/reference/adapters/models).
+ *
+ * For more information about creating and running migrations with Kysely, refer to the [Kysely migrations documentation](https://koskimas.github.io/kysely/#migrations).
+ * ## Naming conventions
+ * If mixed snake_case and camelCase column names is an issue for you and/or your underlying database system, we recommend using Kysely's `CamelCasePlugin` ([see the documentation here](https://koskimas.github.io/kysely/classes/CamelCasePlugin.html)) feature to change the field names. This won't affect NextAuth.js, but will allow you to have consistent casing when using Kysely.
```

Comment:
camel case plugin link is wrong 😆
```suggestion
 * If mixed snake_case and camelCase column names is an issue for you and/or your underlying database system, we recommend using Kysely's `CamelCasePlugin` ([see the documentation here](https://kysely-org.github.io/kysely/classes/CamelCasePlugin.html)) feature to change the field names. This won't affect NextAuth.js, but will allow you to have consistent casing when using Kysely.
```